### PR TITLE
Simple tools for version and flags management

### PR DESCRIPTION
**Pull request by fredboudon**

### CODE REVIEW

**Comment by pradal** [comment]
Composite module names qhould be separated by underscore.
Change names like configdeploy.py into config_deploy.py (or even better configure_meta.py)

File: src/sconsx/config.py [hunk 1]
```diff
@@ -371,7 +372,7 @@ def ALEASolution(options, tools=[], dir=[]):
     if isinstance(platform, Win32):
         # Checking for compiler info first
         compileroptions = deepcopy(options)
-        compilerconf = Config(default_tools,dir)
+        compilerconf = Config([],dir)
```
Comment (pradal) [inline src/sconsx/config.py:375]: This may have a huge effect, isn't it?
How users that depend on compiler will add the compiler? Explicitly?

Comment (pradal) [reply src/sconsx/config.py:375]: Just explain a bit. Thanks

Comment (fredboudon) [reply src/sconsx/config.py:375]: default_tools is already included by default in Config construction.

**Pull request merged**